### PR TITLE
Andrii page

### PR DESCRIPTION
**Pull request by AndriiIv**

### CODE REVIEW

File: src/test/java/page/ChangeLanguage.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+    public WebElement langUa(){
+        return getClickableElement("//a[@class=\"lang__link ng-star-inserted\"]");
+    }
+
+    public WebElement clickBurgerBtn(){
+        return getClickableElement("//button[@class='header__button ng-tns-c94-1']");
+    }
+
+    public WebElement clickChangeLangUA(){
```
Comment (sauronx2) [inline src/test/java/page/ChangeLanguage.java]: Ctrl+Alt+l

File: src/test/java/test/testRozetka/TestLanguage.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        changeLanguage.goToRozetkaUa();
+        changeLanguage.langUa().click();
+
+        Assert.assertTrue(driver.getPageSource().contains("ї"));
+        Assert.assertEquals(getUrl(), "https://rozetka.com.ua/ua/");
+    }
+
+    ////////////burger btn check///////////////////////
+    @Test
+    public void checkLanguageInBurgerBtnUA(){
```
Comment (sauronx2) [inline src/test/java/test/testRozetka/TestLanguage.java]: Ctrl+Alt+l

File: src/test/java/test/testRozetka/TestLanguage.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        ChangeLanguage changeLanguage = new ChangeLanguage(driver);
+        changeLanguage.goToRozetkaUa();
+        changeLanguage.clickBurgerBtn().click();
+        changeLanguage.clickChangeLangUA().click();
+
+        Assert.assertTrue(driver.getPageSource().contains("ї"));
+        Assert.assertEquals(getUrl(), "https://rozetka.com.ua/ua/");
+    }
+
+    @Test
```
Comment (sauronx2) [inline src/test/java/test/testRozetka/TestLanguage.java:42]: Ctrl+Alt+l

File: src/test/java/page/ChangeLanguage.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        return getClickableElement("//div[@class='side-menu__switch ng-tns-c94-1 ng-star-inserted']//a[@class='lang__link ng-star-inserted']");
+    }
+
+    public WebElement clickChangeLangRU(){
+        return getClickableElement("//li[@class=\"lang__item __item __item_state_active ng-star-inserted\"]");
+    }
+
+
+
+
```
Comment (OleksandrMoroz) [inline src/test/java/page/ChangeLanguage.java:38]: Delete space

File: src/test/java/test/testRozetka/TestLanguage.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page.ChangeLanguage;
+import test.AbstractTestInit;
+
+
+public class TestLanguage extends AbstractTestInit {
+
+    @Test
+    public void selectLanguageRU(){
```
Comment (OleksandrMoroz) [inline src/test/java/test/testRozetka/TestLanguage.java:12]: Wrong Test Name
Example: check or get

File: src/test/java/test/testRozetka/TestLanguage.java [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    public void selectLanguageUA(){
+        ChangeLanguage changeLanguage = new ChangeLanguage(driver);
+        changeLanguage.goToRozetkaUa();
+        changeLanguage.langUa().click();
+
+        Assert.assertTrue(driver.getPageSource().contains("ї"));
+        Assert.assertEquals(getUrl(), "https://rozetka.com.ua/ua/");
+    }
+
+    ////////////burger btn check///////////////////////
```
Comment (OleksandrMoroz) [inline src/test/java/test/testRozetka/TestLanguage.java:31]: Delete trash comment

**Review by sauronx2 — APPROVED** [review]

**Pull request merged**